docs/user-guide/api-mediation/api-gateway-configuration.md

```diff
@@ -4,74 +4,128 @@
+         
+    1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
+    2. Find the line that contains the `-Dapiml.service.corsEnabled=false` parameter and set the value to `true`.
+    3. Restart Zowe&trade.
+      
+    Requests through the Gateway now contain a CORS header. 
 
-### apiml.security.auth.provider
+* **apiml.service.hostname**
 
-By default, the API Gateway uses z/OSMF as an authentication provider. It is possible to switch to SAF as the authentication
-provider instead of z/OSMF. The intended usage of SAF as an authentication provider is for systems without z/OSMF.
-If SAF is used and the z/OSMF is available on the system, the created tokens are not accepted by z/OSMF. Use
-the following procedure to switch to SAF. 
+    This property is used to set the API Gateway hostname.
 
-**Follow these steps:**
-     
-1. Open the file `<Zowe install directory>/components/api-mediation/bin/start.sh`.
-2. Find the line that contains the `-Dapiml.security.auth.zosmfServiceId=zosmf` parameter and replace it with `-Dapiml.security.auth.provider=saf`.
-3. Restart Zowe&trade.
+* **apiml.service.port**
+
+    This property is used to set the API Gateway port.
+
+* **apiml.service.discoveryServiceUrls**
+
+    This property specifies the Discovery Service URL used by the service to register to Eureka.
+
+* **apiml.service.preferIpAddress**
+
+    Set the value of this property to `true` if you want to advertise a service IP address instead of its hostname. 
+
+* **apiml.cache.storage.location** 
+
+    This property specifies the location of the EhCache used by Spring.
+
+* **apiml.gateway.timeoutMillis**
 
-Authentication requests now utilize SAF as the authentication provider. API ML can run without zOSMF present on the system. 
+    This property is used to define the timeout value for connection to the services.
 
-### apiml.security.auth.tokenProperties.expirationInSeconds
+* **apiml.security.ssl.verifySslCertificatesOfServices**
 
-This property is relevant only when the JWT token is generated by the API Mediation Layer. API ML generation of the JWT token occurs in the following cases:
+    This parameter makes it possible to prevent server certificate validation.
 
-* z/OSMF is only available as an older version which does not support JWT tokens 
-* The SAF provider is used
- 
-To use a custom configuration for z/OSMF which changes the expiration of LTPA token, it is necessary to also set the expiration in this parameter. 
+    **Important!** Ensure that this parameter is set to `true` in production environments. 
+    Setting this parameter to `false` in production environments significantly degrades the overall security of the system.
 
-**Note:** The default value is 8 hours which mimicks the 8 hour default expiration of the LTPA token in z/OSMF.
+* **apiml.security.auth.zosmfServiceId**
 
-**Follow these steps:**
+    This parameter specifies the z/OSMF service id used as authentication provider.
```

Comment:
Maybe provide the information about the typical default and where it is possible to find it?